src/components/button-group/styles.ts

```diff
@@ -0,0 +1,19 @@
+// import { css } from '@linaria/core'
+import { isMobile } from '@/styles/media'
+import { styled } from '@linaria/react'
+
+export const ElButtonGroup = styled.div`
+  display: flex;
```

Comment:
**question:** Where does the `align-items` CSS come from? Figma?

Reply:
I added that to support mobile-first CSS as there wasn't any in figma.

Reply:
Okay, I don't think this makes sense to centre the buttons in the button group. That would be something the design spec would be clear about if it were required.

Same deal with the `flex-wrap` property. Nothing in the design spec suggests that buttons should wrap within the button group.